services/navigation/builtin/builtin_test.go

```diff
@@ -522,7 +639,7 @@ func TestStartWaypoint(t *testing.T) {
 			}
 
 			// start navigation - set ModeManual first to ensure navigation starts up
-			err = ns.SetMode(ctx, navigation.ModeWaypoint, map[string]interface{}{"experimental": true})
+			err = ns.SetMode(ctx, navigation.ModeWaypoint, nil)
```

Comment:
@biotinker @nicksanford @nfranczak Is this deprecated or just not used yet

Reply:
yep

Reply:
Thank you!